pkg/operator/ceph/object/objectstore.go

```diff
@@ -358,3 +359,12 @@ func GetObjectBucketProvisioner(c *clusterd.Context, namespace string) string {
 	}
 	return provName
 }
+
+func SetBucketProvisionerThreads(c *clusterd.Context) {
```

Comment:
Shouldn't this be part of the bucket code instead?

Reply:
Done